release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -2518,7 +2518,7 @@ In the table below, features are marked with the following statuses:
 // TODO: This known issue should carry forward to 4.8 and beyond! This needs some SME/QE review before being updated for 4.9.
 * In {product-title} 4.1, anonymous users could access discovery endpoints. Later releases revoked this access to reduce the possible attack surface for security exploits because some discovery endpoints are forwarded to aggregated API servers. However, unauthenticated access is preserved in upgraded clusters so that existing use cases are not broken.
 +
-If you are a cluster administrator for a cluster that has been upgraded from {product-title} 4.1 to 4.8, you can either revoke or continue to allow unauthenticated access. It is recommended to revoke unauthenticated access unless there is a specific need for it. If you do continue to allow unauthenticated access, be aware of the increased risks.
```

Comment:
This is an unrelated change to the MetalLB BFD note. If needed, this should be done in a separate PR.

Reply:
OK will remove this I just noticed this kept referring to 4.8 even when release version was rev'd